pkg/resource/hyper_parameter_tuning_job/delta.go

```diff
@@ -110,6 +110,7 @@ func newResourceDelta(
 			delta.Add("Spec.HyperParameterTuningJobName", a.ko.Spec.HyperParameterTuningJobName, b.ko.Spec.HyperParameterTuningJobName)
 		}
 	}
+
```

Comment:
was this automatically added ?

Reply:
Appears so I've rebuilt the code several times and that newline appears